Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -22,7 +22,7 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "BaseTarget.xcconfig"
-#include "WebKitTargetConditionals.xcconfig"
+#include "../../../Configurations/WebKitTargetConditionals.xcconfig"
```

Comment:
It's already included from BaseTarget.xcconfig, so this line is unnecessary.

Reply:
True for all three of these, so I'll made that change in all of them:

```
Source/WebKit/Configurations/WebKit.xcconfig
Source/WebKit/Configurations/WebKitPlatform.xcconfig
Source/WebKit/Configurations/WebKitSwift.xcconfig
```